esp/src/src-react/components/Metrics.tsx

```diff
@@ -227,13 +225,15 @@ export const Metrics: React.FunctionComponent<MetricsProps> = ({
         .maxZoom(Number.MAX_SAFE_INTEGER)
     );
 
+    const [scopeFilter, setScopeFilter] = React.useState("");
     React.useEffect(() => {
         timeline
             .on("click", (row, col, sel) => {
-                setTimelineFilter(sel ? row[7].__hpcc_id : "");
                 if (sel) {
+                    timeline.selection([]);
```

Comment:
I was running into an uncaught exception: timeline.selection is not a function

Reply:
@jeclrsg bumped the @hpcc-js versions...